sprint/start_clean.sh

```diff
@@ -64,6 +64,8 @@ ${GREEN}
 #
 # DB Admin:  ${YELLOW}http://sprint-[ts].ddev.local:8036/${GREEN}
```

Comment:
"DB Admin" sounds awfully stiff and formal, somewhere else I changed this to "phpMyAdmin:"

Reply:
Updated. I needed to change the spaces in all the lines by an additional space.